vignettes/bds_finding.Rmd

```diff
@@ -611,7 +766,7 @@ ad_advs.R | Derive a new PARAMCD based on existing PARAMCDs (MAP, based on Systo
 Example 1 (Creating a new record):
```

Comment:
Could you please set it at level 3: ###?
Also I would add more details on what record is added. like: In this example, a new record is created or each subject and parameter: AVAL is set to the last value on treatment, AVISIT is set to "End of Treatmenet " and AVISISTN...